specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2022-05-15/healthcare-apis.json

```diff
@@ -3669,7 +3673,7 @@
         },
         "corsConfiguration": {
           "description": "Fhir Service Cors configuration.",
-          "$ref": "#/definitions/FhirServiceCorsConfiguration"
```

Comment:
Unfortunately, we can't make this change.  It causes breaking changes with the SDKs.

Reply:
> Unfortunately, we can't make this change. It causes breaking changes with the SDKs.

Thank you for pointing this out. Updated,